virt/virt-2-5-release-notes.adoc

```diff
@@ -101,6 +101,7 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 //CNV-7318 - Open the vnc/serial console in a new window
 
 //CNV-7169 - Auto-clone base images when creating VM in UI
+You can now create default OS images and automatically upload them using the {product-title} Web Console. A _default OS image_ is a bootable disk containing an operating system and all of the operating system's configuration settings, such as drivers. You use a default OS image to create bootable virtual machines with specific configurations.
```

Comment:
It looks like this should be a bulleted item, to match the ones around it.

Reply:
Also:

s/Web Console/web console/

To match usage elsewhere in the repo.

Reply:
Done.